GoogleTestAdapter/Core/Resources.resx

```diff
@@ -290,6 +322,182 @@
+    <comment>{0} represents a command-line flag.</comment>
+  </data>
+  <data name="OptionShowReleaseNotes" xml:space="preserve">
+    <value>Show release notes after update</value>
+  </data>
+  <data name="OptionShowReleaseNotesDescription" xml:space="preserve">
+    <value>If true, a dialog with release notes is shown after the extension has been updated.</value>
+  </data>
+  <data name="OptionShuffleTests" xml:space="preserve">
+    <value>Shuffle tests per execution</value>
+  </data>
+  <data name="OptionShuffleTestsDescription" xml:space="preserve">
+    <value>If true, tests will be executed in random order. Note that a true randomized order is only given when executing all tests in non-parallel fashion. Otherwise, the test excutables will most likely be executed more than once - random order is than restricted to the according executions.
+Google Test option: {0}</value>
+    <comment>{0} represents a command-line flag.</comment>
+  </data>
+  <data name="OptionShuffleTestsSeed" xml:space="preserve">
+    <value>Shuffle tests: Seed</value>
+  </data>
+  <data name="OptionShuffleTestsSeedDescription" xml:space="preserve">
+    <value>0: Seed is computed from system time, 1&lt;=n&lt;={0}: The given seed is used. See note of option '{1}'.
+Google Test option: {2}</value>
+    <comment>{0} is an integer representing the maximum value of a seed. {1} represents the localized name of another option. {2} represents a command-line flag.</comment>
+  </data>
+  <data name="OptionTestDiscoveryRegex" xml:space="preserve">
+    <value>Regex for test discovery</value>
+  </data>
+  <data name="OptionTestDiscoveryRegexDescription" xml:space="preserve">
+    <value>If non-empty, this regex will be used to discover the Google Test executables containing your tests.
+Default regex:</value>
+    <comment>Followed by a regular expression</comment>
+  </data>
+  <data name="OptionTestDiscoveryTimeoutInSeconds" xml:space="preserve">
+    <value>Test discovery timeout in s</value>
+    <comment>"s" stands for seconds</comment>
+  </data>
+  <data name="OptionTestDiscoveryTimeoutInSecondsDescription" xml:space="preserve">
+    <value>Number of seconds after which test discovery will be assumed to have failed. 0: Infinite timeout</value>
+  </data>
+  <data name="OptionTestNameSeparator" xml:space="preserve">
+    <value>Test name separator</value>
+  </data>
+  <data name="OptionTestNameSeparatorDescription" xml:space="preserve">
+    <value>Test names produced by Google Test might contain the character '/', which makes VS cut the name after the '/' if the test explorer window is not wide enough. This option's value, if non-empty, will replace the '/' character to avoid that behavior. Note that '\', ' ', '|', and '-' produce the same behavior ('.', '_', ':', and '::' are known to work - there might be more). Note also that traits regexes are evaluated against the tests' display names (and must thus be consistent with this option).</value>
```

Comment:
1. Avoid "VS" abbreviation.
2. Capitalize Test Explorer
3. I would remove "there might be more".